src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS.UnitTests/ProjectSystem/VS/Properties/CSharpProjectDesignerPageProviderTests.cs

```diff
@@ -24,6 +25,24 @@ public void Constructor_DoesNotThrow()
             var pages = await provider.GetPagesAsync();
 #pragma warning restore RS0003 // Do not directly await a Task
 
+            Assert.Equal(pages.Count(), 6);
+            Assert.Same(pages.ElementAt(0), CSharpProjectDesignerPage.Application);
+            Assert.Same(pages.ElementAt(1), CSharpProjectDesignerPage.Build);
+            Assert.Same(pages.ElementAt(2), CSharpProjectDesignerPage.BuildEvents);
+            Assert.Same(pages.ElementAt(3), CSharpProjectDesignerPage.Debug);
+            Assert.Same(pages.ElementAt(4), CSharpProjectDesignerPage.ReferencePaths);
+            Assert.Same(pages.ElementAt(5), CSharpProjectDesignerPage.Signing);
+        }
+
+        [Fact]
+        public async Task GetPagesAsync_WithPackCapability()
+        {
+            var provider = CreateInstance(ProjectCapability.Pack);
+
+#pragma warning disable RS0003 // Do not directly await a Task (see https://github.com/dotnet/roslyn/issues/6770)
```

Comment:
This should use bug number https://github.com/dotnet/roslyn-project-system/issues/124. Also, this is in test code, right? Shouldn't this warning already be disabled for our test code?